RNA-seq/setup/setup.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# This script runs two snakemake workflows to populate the 
+# `/shared/training-modules/RNA-seq` directory with preprocessed data
+# for the notebooks in that module
+
+
+# safety flags
+set -euo pipefail
+# Set the working directory to the directory of this file
+cd "$(dirname "${BASH_SOURCE[0]}")"
+
+jobs=12
```

Comment:
Checking understanding question: This is basically available cores, right?

Reply:
Yes. Or more accurately how many you want to allow.